### PR TITLE
build: add support for node 16

### PR DESCRIPTION
**Pull request by iamsobanjaved**
Adding support for node 16

### Screenshots:

Courses page:

Screenshot 2022-04-20 at 11 59 08 PM

No error reported on console:

Screenshot 2022-04-20 at 11 59 30 PM

### CODE REVIEW

**Review by UsamaSadiq — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -43,14 +44,13 @@ jobs:
           TESTNAME: quality
           TARGETS: "quality"
       - name: test js
-        run: ./.github/scripts/testing.sh
+        run: ./.github/scripts/testing-js.sh
```
Comment (awais786) [inline .github/workflows/ci.yml:47]: node16 is brining new translations. One solution might be merge this PR and then run transifex job and then enable this check again.

**Review by awais786 — APPROVED** [review]

**Comment by schenedx** [comment]
I reviewed all and my biggest concern is on the upgrade of `babel` libraries upgrading to `@babel` versions.
Have you tried to run insights dashboard frontend using this branch? Are the UI impacted by looking at it?

**Comment by iamsobanjaved** [comment]
@schenedx I have added screenshots in the description of the PR, you can verify if UI is fine by these pics. As we have upgraded babel and changed to `@babel`, this was causing issues that delayed this. I forgot to update the name of one of the babel plugins in the `webpack` config. But as now build command is working after that fix, I think we are safe now.

**Review by schenedx — APPROVED** [review]
@iamsobanjaved Alright... Let's try this. 👍 
Feel free to merge. I can test on stage as well.

**Comment by awais786** [comment]
@schenedx stage is looking fine.
https://stage-insights.edx.org/

**Comment by schenedx** [comment]
@awais786 Yes. It looks like this is successful. Want to go ahead and deploy the changes to prod?

**Comment by awais786** [comment]
@schenedx does this pr and config PR need to backport into nutmeg ?

**Pull request merged**